.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
> so I suppose it finish immediately (it is stopped) and doesn't continue to its normal end

Wrong. This just change the Travis status (updated quicker). But **all the jobs** will be finished even if `fast_finish` enabled or not. :+1: